modules/navier_stokes/src/auxkernels/WallFunctionWallShearStressAux.C

```diff
@@ -98,31 +98,30 @@ WallFunctionWallShearStressAux::computeValue()
   const auto state = determineState();
 
   // Get the velocity vector
-  ADRealVectorValue velocity(_u_var->getElemValue(&elem, state));
+  RealVectorValue velocity(_u_var->getElemValue(&elem, state).value());
```

Comment:
```suggestion
  RealVectorValue velocity(MetaPhysicL::raw_value(_u_var->getElemValue(&elem, state)));
```
Let's try to be consistent